src/sentry/sentry_metrics/querying/data_v2/execution.py

```diff
@@ -560,13 +618,23 @@ def schedule(
 
         Note that this method won't execute the query, since it's lazy in nature.
         """
-        executable_query = ExecutableQuery(
-            with_series=True,
-            with_totals=True,
-            # We identify the query with its index.
-            identifier=str(len(self._scheduled_queries)),
+        # For now, we are always building a (totals -> series) query, but the execution engine is fully capable of
+        # supporting either a single totals or series query.
+        executable_query = ScheduledQuery(
```

Comment:
We always build `totals -> series` queries for now, but the engine is fully capable of allowing even single `totals` or `series` queries. It was designed to be flexible to allow us to make more efficient queries in case we need to query only one of those and we do not need `limit` nor `order`.